test/integration/targets/ec2_spot_pricing_history_facts/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+# ============================================================
+# Test by providing all the parameters
+- name: Set up AWS connection info
+  set_fact:
+    aws_connection_info: &aws_connection_info
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      region: "{{ aws_region }}"
```

Comment:
I've updated our CI system to allow the actions required by this PR. You'll need one additional change:

```yaml
security_token: "{{ security_token }}"
```